### PR TITLE
Do not throw a notice when using Airbrake in CLI

### PR DESCRIPTION
**Pull request by tmotyl**
Resolves https://github.com/airbrake/Airbrake-Magento/issues/9

### CODE REVIEW

File: app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php [hunk 1]
```diff
@@ -11,9 +11,15 @@ public function __construct() {
         require_once Mage::getBaseDir('lib') . '/Airbrake/Configuration.php';
 
         $options = array();
-        $requestUri = explode("/",$_SERVER['REQUEST_URI']);
-        $options['action'] = array_pop($requestUri);
-        $options['component'] = implode('/',array_slice($requestUri,-2));
+        if (isset($_SERVER['REQUEST_URI'])) {
```
Comment (kyrylo) [inline app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php]: Would you mind adding a simple comment explaining why this check is needed?

Comment (tmotyl) [inline app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php]: REQUEST_URI is not available in the CLI (console) context

Comment (kyrylo) [inline app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php]: It would be nice to add this to the code as a comment.

**Comment by tmotyl** [comment]
done

**Comment by kyrylo** [comment]
:+1:

**Pull request merged**